foreman_remote_execution_core.gemspec

```diff
@@ -22,4 +22,5 @@ DESC
   s.add_runtime_dependency('ed25519')
   s.add_runtime_dependency('foreman-tasks-core', '>= 0.3.1')
   s.add_runtime_dependency('net-ssh')
+  s.add_runtime_dependency('smart_proxy_remote_execution_ssh', '>= 0.4.0')
```

Comment:
doesn't this make us still release the SPREX before this one anyway?

Reply:
It makes us release sprex before rex_core, but non-core rex can be released independently as it can work with both old and new rex_core